### PR TITLE
Initial support for csv export for metadata

### PR DESCRIPTION
**Pull request by drewpotter**
You can now download a CSV file by going to `/download` in the address bar. Currently there is no button for this, but the download works and has been tested.

### CODE REVIEW

File: app/models/metadata.rb [hunk 1]
```diff
@@ -58,4 +58,33 @@ def self.metadata_to_json
...
+    metadata_columns = Metadata.new.attributes.keys
+    metadata_columns.delete("created_at")
+    metadata_columns.delete("updated_at")
+
+    metadata_columns.map! { |e|
+      e.gsub("_", " ").capitalize
+    }
+
+    csv << metadata_columns.join(',')
+    csv << "\r\n"
```
Comment (Levia) [inline app/models/metadata.rb]: You probably don't need the `\r`. Same below

File: config/routes.rb [hunk 2]
```diff
@@ -9,4 +9,6 @@
   get 'environmental-impact-assessment', to: 'environmental_impact_assessment#index'
   get 'ecosystem-assessment', to: 'ecosystem_assessment#index'
   get 'ecosystem-services', to: 'ecosystem_services#index'
+  get '/download', to: 'metadata#download'
```
Comment (Levia) [inline config/routes.rb]: I would prefer to have `metadata/download` as a path, so it's clear what the download refers to.

File: app/models/metadata.rb (hunk 1 above)
Comment (drewpotter) [reply app/models/metadata.rb]: Thanks I have fixed this.

File: config/routes.rb (hunk 2 above)
Comment (drewpotter) [reply config/routes.rb]: Thanks I have fixed this too.

**Pull request merged**